Assets/Scripts/Player/PlayerController.cs

```diff
@@ -91,4 +105,50 @@ public class PlayerController : MonoBehaviour {
             transform.rotation = Quaternion.Euler(0f, angle, 0f);
         }
     }
+    
+    private void ExecuteLadderMove() {
+        if (Input.GetButtonDown("Jump")) {
+            DisengageLadder();
+            verticalSpeed = Vector3.up * jumpHeight;
+            controller.Move(verticalSpeed * Time.deltaTime);
```

Comment:
Is this `controller.Move` actually necessary? I'd think ExecuteMove would run almost immediately afterwards which does the same